make_dashboards.py

```diff
@@ -248,11 +249,36 @@ def write_as_file(name_path, result, dir):
     name = os.path.basename(name_path)
     write_json(os.path.join(dir, name), result["dashboard"])
 
+def make_kubernetes_operator(result):
+    kubernetes_node_variable = {
+            u'class': u'template_variable_all',
+            u'label': u'kubernetes_node',
+            u'name': u'kubernetes_node',
+            u'query': u'label_values(node_disk_writes_completed, instance)'
+            }
+
+    result["dashboard"]["templating"]["list"].append(dict(kubernetes_node_variable))
+
+    for row in result['dashboard']['rows']:
+        if row["class"] == "row":
+           for panel in row['panels']:
+               if 'targets' in panel:
+                   for target in panel['targets']:
+                       if '(node_' in target['expr']:
```

Comment:
First, this patch is too specific, you are taking metrics specific names (node_) that you know belongs to the node_exporter.
I would rather not do that.
Second, I'm trying to understand, why do you need all that?
wouldn't it all work out of the box if you specify node_exporter file with kubernetes nodes in it?

Reply:
For your first point: me too. Open to suggestions.

Second: 
Are you talking about the scrape config? If you are, there is a configuration file for it and it does scrape the `kubernetes_node`variable.

Grafana doesn't use `kubernetes_node` nor cares. I can't override `node` to `kubernetes_node`, because, afaik, you can't have two labels with the same name. The `instance=[[node]]` query won't select anything because `instance` uses kubernetes node's internal IP instead of Scylla's internal IP which won't match the `node` definition (which is Scylla's internal IP). Plus, on the dashboards, `node_` variables seems to care about `dc`, which doesn't exist on kubernetes. 

For example, metrics:
```
node_*(instance=10.0.0.1, kubernetes_node=gke.node1.cluster)
scylla_*(instance=50.0.0.1, dc=as, ....)
```
Label:
```
node=label_target(scylla_*, instance) // so, 50.0.0.x
```
Query:
```
node_*(instance=[[node]], dc=$dc, ...) // nothing
```

What this patch does:
```
node_*(instance=10.0.0.1, kubernetes_node=gke.node1.cluster)
scylla_*(instance=50.0.0.1, dc=as, ....)
```
Label:
```
node=label_target(scylla_*, instance) // so, 50.0.0.x
kubernetes_node=label_target(node_*, kubernetes_node) // so, 10.0.0.x
```
Query:
```
node_*(instance=[[kubernetes_node]],  dc=$dc|$^, ...) // something
```